### PR TITLE
New feature name

### PR DESCRIPTION
**Pull request by gogr**

### CODE REVIEW

File: ui_tests/src/test/resources/com/saucedemo/account/signIn.feature [hunk 1]
```diff
@@ -6,12 +6,10 @@ Feature: Sign In
     When I set 'standard_user' as username on Sign in page
     And I set 'secret_sauce' as password on Sign in page
     And I click 'Sign In' button on Sign in page
-    Then I am on Products page
```
Comment (gogr) [inline ui_tests/src/test/resources/com/saucedemo/account/signIn.feature]: This line shouln't be removed

File: ui_tests/src/test/resources/com/saucedemo/account/signIn.feature [hunk 2]
```diff
@@ -6,12 +6,10 @@ Feature: Sign In
...
-
   @signIn
   Scenario: Sign in with invalid password
     Given I open start page
     When I set 'standard_user' as username on Sign in page
-    And I set 'thisIsSomeInvalidPassword' as password on Sign in page
+    And I set 'thisIsSomeInvasdfvdflidPassword' as password on Sign in page
     And I click 'Sign In' button on Sign in page expecting failure
```
Comment (gogr) [inline ui_tests/src/test/resources/com/saucedemo/account/signIn.feature]: Upates required

File: ui_tests/src/test/resources/com/saucedemo/account/signIn.feature (hunk 1 above)
Comment (gogr) [reply ui_tests/src/test/resources/com/saucedemo/account/signIn.feature]: Done

File: ui_tests/src/test/resources/com/saucedemo/account/signIn.feature (hunk 2 above)
Comment (gogr) [reply ui_tests/src/test/resources/com/saucedemo/account/signIn.feature]: Done

File: ui_tests/src/test/resources/com/saucedemo/account/signIn.feature (hunk 1 above)
Comment (ahliddinsalim) [reply ui_tests/src/test/resources/com/saucedemo/account/signIn.feature]: cool

**Pull request closed without merging**